packages/appkit/src/client/appkit-base-client.ts

```diff
@@ -1791,7 +1791,27 @@ export abstract class AppKitBaseClient {
     OptionsController.setManualWCControl(Boolean(this.options?.manualWCControl))
     this.universalProvider =
       this.options.universalProvider ?? (await UniversalProvider.init(universalProviderOptions))
-    // Clear the session if we don't want to reconnect on init
+
+    const originalDisconnect = this.universalProvider.disconnect.bind(this.universalProvider)
```

Comment:
Weird, why we need this? can you add jsdoc since this workaround seems someting extraordinary